client/state/ui/selectors.js

```diff
@@ -6,7 +6,7 @@
  */
 export function getSelectedSite( state ) {
 	if ( ! state.ui.selectedSiteId ) {
-		return null;
+		return false;
```

Comment:
I can only speculate here, but if you don't take the `getSelectedSite( state ).attribute` use case into account, defaulting to `null` for no-object might be somewhat more consistent. Also, keep in mind that this selector isn't used widely yet (the only other occurrence being in `client/post-editor/editor-sharing/container.jsx`), so I'd assume that there hasn't been any use for signalling that no site/all sites are selected (which would also explain why the `ui` reducer wasn't updated [here](https://github.com/Automattic/wp-calypso/pull/2657/files#diff-f11e4a0f3fa89aa5176d3244e323a240R130) before).